driver/syscall_table.c

```diff
@@ -177,36 +177,12 @@ const struct syscall_evt_pair g_syscall_table[SYSCALL_TABLE_SIZE] = {
 #ifdef __NR_fcntl64
 	[__NR_fcntl64 - SYSCALL_TABLE_ID0] =                    {UF_USED, PPME_SYSCALL_FCNTL_E, PPME_SYSCALL_FCNTL_X, PPM_SC_FCNTL64},
 #endif
-/* [__NR_old_select - SYSCALL_TABLE_ID0] =	{UF_USED, PPME_GENERIC_E, PPME_GENERIC_X}, */
-	[__NR_pselect6 - SYSCALL_TABLE_ID0] =                   {UF_USED | UF_ALWAYS_DROP, PPME_GENERIC_E, PPME_GENERIC_X, PPM_SC_PSELECT6},
```

Comment:
Cleaned up multiple syscalls that were mapped to an `UF_ALWAYS_DROP` generic event; it behaves the same as using the widely used ` { .ppm_sc = ... }` format; but the latter format is also recognized by syscalls-bumper as "syscalls mapped to generic event".

Note: the UF_USED is used in drivers like:
```
if (sc_evt->flags & UF_USED) {
		evt_type = sc_evt->enter_event_type;
		drop_flags = sc_evt->flags;
	} else {
		evt_type = PPME_GENERIC_E;
		drop_flags = UF_ALWAYS_DROP;
	}
```
(kmod behaves the same).
As you can see, having UF_USED and then a generic event with `UF_ALWAYS_DROP` is exactly the same as having the default `UF_UNUSED`.